src/cobra/io/web/bigg_models_repository.py

```diff
@@ -0,0 +1,84 @@
+"""Provide a concrete implementation of the BioModels repository interface."""
+
+
+from io import BytesIO
+from typing import Union
+
+import httpx
+
+from .abstract_model_repository import AbstractModelRepository
+
+
+class BiGGModels(AbstractModelRepository):
+    """
+    Define a concrete implementation of the BiGG Models repository.
+
+    Attributes
+    ----------
+    name : str
+        The name of the BiGG Models repository.
+
+    """
+
+    name: str = "BiGG Models"
+
+    def __init__(
+        self,
+        *,
+        url: Union[httpx.URL, str] = "http://bigg.ucsd.edu/static/models/",
+        **kwargs,
+    ):
+        """
+        Initialize a BiGG Models repository interface.
+
+        Parameters
+        ----------
+        url : httpx.URL or str, optional
```

Comment:
Why should you be able to set this? This should be constant and immutable for BIGG. Otherwise this would provide a security risk where you can make it look like you're downloading from BIGG even if you aren't.

Reply:
Fair point. Just a habit in case someone hosts their own BiGG database or the URL has changed and we haven't caught up yet (or new URL with old code version).

Reply:
I was thinking, though, you can host your own on-premise installation of BiGG so maybe that's a reason to keep it flexible?